src/padding.rs

```diff
@@ -0,0 +1,99 @@
+use crate::CellOption;
+use papergrid::{Entity, Grid, Settings};
+
+/// Padding is responsible for a left/right/top/bottom indent of particular and fill one set characters.
+///
+/// ```rust,no_run
+///   # use tabled::{Style, Padding, Row, Table, Modify};
+///   # let data: Vec<&'static str> = Vec::new();
+///     let table = Table::new(&data).with(Modify::new(Row(..1)).with(
+///                             Padding::new(0, 0, 1, 1)
+///                                 .set_left_fill('>')
+///                                 .set_right_fill('<')
+///                                 .set_bottom_fill('^')
+///                                 .set_top_fill('V')));
+/// ```
+#[derive(Debug, Clone, Copy)]
+pub struct Padding {
+    pub top: Indent,
+    pub bottom: Indent,
+    pub left: Indent,
+    pub right: Indent,
+}
```

Comment:
As you've mentioned there might be no need to duplicate these structures from `papergrid`.

So we could create a wrapper.

```rust
pub struct Padding(papergrid::Padding);
```

Or use a type definition

```rust
pub type Padding = papergrid::Padding
```

Reply:
I decided to use wrapper 👉🏻 type alias not allow to use `impl`  
```
Cannot define inherent `impl` for a type outside of the crate where the type is defined
```